content/docs/command-reference/params/index.md

```diff
@@ -16,62 +16,70 @@ positional arguments:
 
 ## Description
 
-In order to track parameters and hyperparameters associated to machine learning
-experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. They usually have simple names like `epochs`,
-`learning-rate`, `batch_size`, etc.
+Parameters are simple values used inside your code to influence the results
+(e.g. machine learning [hyperparameters]). DVC can track these as key/value
+pairs from structured YAML 1.2, JSON, TOML 1.0,
+[or Python](#examples-python-parameters-file) files (`params.yaml` by default),
+from which your code should also read them.
 
-To start tracking parameters, list them under the `params` field of `dvc.yaml`
-stages (manually or with the the `-p`/`--params` option of `dvc stage add`). For
-example:
+Multiple stages of a <abbr>pipeline</abbr> can [use the same params file] as
+<abbr>dependency</abbr>, but only certain values will affect each
+<abbr>stage</abbr>. Params usually have simple names like `epochs`,
+`learning-rate`, `batch_size`, etc. Example:
+
+```yaml
+epochs: 900
+tuning:
+  - learning-rate: 0.945
+  - max_depth: 7
+```
+
+To start tracking parameters, list their names under the `params` field of
+`dvc.yaml` (manually or with the the `-p`/`--params` option of `dvc stage add`).
+For example:
 
 ```yaml
 stages:
   learn:
-    cmd: ./deep.py
+    cmd: python deep.py
     params:
-      - epochs # track specific parameter (from params.yaml)
-      - tuning.learning-rate
-      - myparams.toml: # track specific params from custom file
-          - batch_size
-      - config.json: # track all parameters in this file
+      - epochs # specific param from params.yaml
+      - tuning.learning-rate # from params.yaml
+      - myparams.toml:
+          - batch_size # specific param from custom file
+      - config.json: # all params in this file
```

Comment:
Should we have an example that is a key for a section of params, like `- train` or `- model_params`, especially since we no longer mention anything about trees/dicts/hierarchies?

Reply:
Good point. Adding ⌛